### PR TITLE
Concat txts from the same chat

### LINKED ISSUE #19: Add a function to merge two txt by dates

### PR DESCRIPTION
**Pull request by mayamz**
Based on PR #22 that parse all formats with one envelope function.

### CODE REVIEW

File: defaults.py [hunk 1]
```diff
@@ -24,4 +24,11 @@
 }
 TIMEZONE_SHIFT = datetime.timedelta(hours=0)  # Thanks Whatsapp
 
-HEBREW_LETTERS = "[אבגדהוזחטיכלמנסעפצקרשתךםןףץ]"
\ No newline at end of file
+HEBREW_LETTERS = "[אבגדהוזחטיכלמנסעפצקרשתךםןףץ]"
+
+ANDROID_DATES = r'(\d{2}\/\d{2}\/\d{4}\, \d{1,2}:\d{2}) \- (.*?): (.*)$'
```
Comment (TrueMichato) [inline defaults.py:29]: We need to check if this also covers phones who use AM/PM notations on hours - I suspect it won't, and that we will need to devise some method to differentiate

File: read_chat.py
Comment (TrueMichato) [inline read_chat.py:1]: Just wanted to say this looks great 🥇

File: whatsapp statistics.ipynb
Comment (TrueMichato) [inline whatsapp statistics.ipynb:1]: You know, you would think github would know how to handle jupyter notebooks better, but it really doesn't help me here.

File: defaults.py (hunk 1 above)
Comment (mayamz) [reply defaults.py:29]: Agree. Adding as separate issue. I will need to do experiments with the settings to see if it changes the text retrospectively.

File: whatsapp statistics.ipynb
Comment (mayamz) [reply whatsapp statistics.ipynb:1]: No they're awful with that. The way I found to review a notebook is to open the branch and run it, then in github find the relevant row and comment.

**Review by TrueMichato — APPROVED** [review]

**Pull request merged**